### DISCUSSION TITLE
adjusting the camera pivot

### DISCUSSION THREAD

**Discussion by SamTheRuppellsFox** [xTracr/RealCamera#49] [Q&A] [answered]
I have a non-vanilla character model, which causes issues even when I adjust the camera height offset. It probably caused by the head pivot point and the camera pivot being misaligned. Is there any way to change the camera pivot point's XYZ coordinates?

**Comment by xTracr** [comment]
Are you able to explain the situation in more detail, such as what kind of effect you want and what the pivot point of the camera actually looks like in game. 
Because I don't seem to have had the problem you describe when using it.

**Reply by SamTheRuppellsFox** [reply]
So, the pivot point is the point around which the object rotates. In my custom character's case the head pivot is 1/8th of a block higher than normal to make it work with the increased model height which causes the camera to clip into the model's head when I look up

https://github.com/xTracr/RealCamera/assets/134311972/bff48185-02d8-4dab-8037-a30bc5ede769

![image](https://github.com/xTracr/RealCamera/assets/134311972/904db9fe-3207-4bc2-bbc4-3d586b7e2e55)
(normal model for comparison)

**Comment by xTracr** [comment, marked as answer]
This should be caused by not binding correctly to where you want.  
When it works correctly, it should look like this:  

https://github.com/xTracr/RealCamera/assets/57320980/934ca9d1-817e-4243-9a0c-83b1af9d06af
